boards/nxp/frdm_mcxn947/frdm_mcxn947.dtsi

```diff
@@ -61,6 +61,12 @@
 	clock-frequency = <I2C_BITRATE_STANDARD>;
 };
 
+&flexcomm2_lpuart2 {
+	current-speed = <115200>;
+	pinctrl-0 = <&pinmux_flexcomm2_lpuart>;
+	pinctrl-names = "default";
+};
```

Comment:
Should this be in the test overlay file as well.

Reply:
I've updated the PR to enable LPUART2 for I2C and UART mode simultaneously, since this is a feature of the IP. Given this, I think we can keep this in the board file since it does not conflict with the LPI2C2 pins